src/main/resources/templates/add-crime-form.html

```diff
@@ -26,7 +26,8 @@ <h2>Add crime</h2>
         <div class="col-sm-8">
             <select class="form-control" th:field="*{address}">
                 <option selected th:value="0"></option>
-                <option th:each="address: ${listAddresses}" th:text="${address.city}" th:value="${address.id}"></option>
+                <option th:each="address: ${listAddresses}" th:text="${address.city + ', ' + address.zipCode +
+                ', ' + address.streetAddress}" th:value="${address.id}"></option>
```

Comment:
```suggestion
                <option th:each="address: ${listAddresses}" th:text="${address.streetAddress + ', ' + address.zipCode + ', ' + address.city}" th:value="${address.id}"></option>
```

Suggest we start with Address and end with city